sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -182,17 +182,22 @@ private static void checkCompatibleNodeVersion(String nodeExecutable) {
 
   private static Map<String, List<InputFile>> getInputFileByTsconfig(Iterable<InputFile> inputFiles, File projectBaseDir) {
     Map<String, List<InputFile>> inputFileByTsconfig = new HashMap<>();
-
+    int filesWithoutTSConfig = 0;
     for (InputFile inputFile : inputFiles) {
       File tsConfig = findTsConfig(inputFile, projectBaseDir);
       if (tsConfig == null) {
+        filesWithoutTSConfig++;
         String message = "No tsconfig.json file found for [%s] (looking up the directories tree until project base directory [%s]). Using default configuration.";
-        LOG.warn(String.format(message, inputFile.uri(), projectBaseDir.getAbsolutePath()));
+        LOG.debug(String.format(message, inputFile.uri(), projectBaseDir.getAbsolutePath()));
         inputFileByTsconfig.computeIfAbsent(DEFAULT_TSCONFIG, x -> new ArrayList<>()).add(inputFile);
       } else {
         inputFileByTsconfig.computeIfAbsent(tsConfig.getAbsolutePath(), x -> new ArrayList<>()).add(inputFile);
       }
     }
+    if (filesWithoutTSConfig > 0) {
```

Comment:
you don't need counter, you can just `inputFileByTsconfig.get("DEFAULT_TSCONFIG")`